app/services/PaymentService.scala

```diff
@@ -48,11 +49,18 @@ class PaymentService(val stripeService: StripeService) {
     }
   }
 
+  def identityIdForAccount(purchaserIds: PurchaserIdentifiers) = {
+    purchaserIds.identityId match {
+      case Some(idUser) => idUser.id
+      case None => ""
```

Comment:
ok paul mentioned that this is only a temporary state until `purchaserIds.identityId` becomes non optional in a few days